shared/scss/base/base.scss

```diff
@@ -315,6 +315,21 @@ a {
         }
 
     }
+    
+    &.icon__error {
+        width: 24px;
+        height: 24px;
+        vertical-align: bottom;
+
+        .is-browser--chrome & {
+            background-image: url($icon__error--chrome);
+        }
+
+        .is-browser--moz & {
+            background-image: url($icon__error--moz);
+        }
```

Comment:
What do you think about using `@include cross_platform_bg_image("error.svg");` instead?

I've wanted to suggest changing all the places where background images are added using the `cross_platform_bg_image` mixin.

My thinking is:
- the mixin is one line in one file vs 8 lines across two files with the old method
- the compiled CSS looks exactly the same